DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: civis
 Title: R Client for the 'Civis Platform API'
-Version: 3.1.2
+Version: 3.1.3
 Authors@R: c(
-  person("Civis", "Support", email = "support@civisanalytics.com", role = "cre"),
+  person("Dustin", "Leatherman", email = "dleatherman@civisanalytics.com", role = "cre"),
```

Comment:
We need an actual person here. I put myself so I can at least review the emails but it might be more appropriate to have @mattyb as the point of contact.

Reply:
Sounds good. I'll stay as the contact for now.